pkg/kubelet/dockertools/docker_test.go

```diff
@@ -734,6 +739,14 @@ func TestMakePortsAndBindings(t *testing.T) {
 			if value[0].HostIP != "" {
 				t.Errorf("Unexpected host IP: %s", value[0].HostIP)
 			}
+		case "446":
```

Comment:
yep, I'll rewrite a test to check 1-to-N mapping instead of patch the existing one, thanks for your comments :)